docs: Restrict example AWS CloudWatch IAM policy, add Vercel setup notes (#15)
dacbd/vercel-log-drain: README.md

| `--loki-basic-auth-user` | `VERCEL_LOG_DRAIN_LOKI_USER`         | `""`          | Loki basic auth username                 |
| `--loki-basic-auth-pass` | `VERCEL_LOG_DRAIN_LOKI_PASS`         | `""`          | Loki basic auth password                 |

## Setting up

Vercel requires that you host the application over HTTP or HTTPS, and have it be accessible from the public internet.

> `vercel-log-drain` itself only supports HTTP – but you can put a HTTPS load balancer in front of it.

Reviewer: ```suggestion
`vercel-log-drain` itself only supports HTTP – so you should put an HTTPS load balancer in front of it.
```

To add new log drains in Vercel:

1. Go to the [Vercel account dashboard](https://vercel.com/account).
2. Find the team to configure, and click `...` → Manage
3. Select Log Drains

The parameters you'll need for `vercel-log-drain` are:

* Delivery format: JSON
* Custom secret: the Vercel secret you set with `--vercel-secret` or `VERCEL_SECRET`
* Endpoint: `https://${your_hostname}/vercel`

> * Custom headers: add a random secret header value, and configure your load balancer to require that header

Reviewer: ```suggestion
* (optional) Custom headers: add a random secret header value, and configure your load balancer to require that header (to help filter out bot noise)
```
